Google.PowerShell/PubSub/GcpsMessage.cs

```diff
@@ -194,20 +194,19 @@ protected override void ProcessRecord()
                 IList<string> returnedMessageIds = response.MessageIds;
                 if (returnedMessageIds.Count != requestBody.Messages.Count)
                 {
-                    throw new InvalidOperationException($"Only {returnedMessageIds.Count} out of {requestBody.Messages.Count} is published.");
+                    throw new InvalidOperationException($"Only {returnedMessageIds.Count} out of {requestBody.Messages.Count} published.");
                 }
 
                 for (int index = 0; index < returnedMessageIds.Count; index += 1)
                 {
                     PubsubMessage publishedMessage = requestBody.Messages[index];
-                    publishedMessage.MessageId = returnedMessageIds[0];
```

Comment:
Should there be some more intrusive tests here?  I'm surprised neither of these  caused issues.